docs/Installation/Upgrading.md

```diff
@@ -20,7 +20,8 @@
     - Any themes or plugins you had installed. The bundled themes (Thanks Roy, Berlin, Seasons) and plugins (COinS, Exhibit Builder, Simple Pages) will already have been updated with Omeka, so don't move them back over.
     - If you've modified any of Omeka's core files, you'll need to move those changes over too. Be careful with this and make your edits line by line, as the original file may have been changed in newer versions. 
 1.  Finally, you may need to **upgrade your database**. Go to your Omeka site's administrative panel in your web browser.
-    - If a database upgrade is needed, you'll automatically be redirected to an upgrade page (`example.com/admin/upgrade`). Click "Upgrade" to update your database to reflect the new data model and finish the upgrade. 
+    - To access your site's administrative panel after upgrading, go to `your-site-url/admin`. If a database upgrade is needed, you will not be able to access your site's dashboard from your site's home page, which will display the message, "Public site is unavailable until the upgrade completes."
```

Comment:
Sorry, for consistency this should be `example.com/admin` not `your-site-url/admin`.